### DISCUSSION TITLE
Data sharing?

### DISCUSSION THREAD

**Discussion by ChrisKeefe** [ianyh/Amethyst#1308] [Q&A] [answered]
Hi @ianyh!
Does Amethyst share any data with external servers, or does it/can it run on the local machine only? My employer's IT policy is pretty restrictive of software that shares any data outside of the organization.

Thanks,
Chris

**Comment by ianyh** [comment, marked as answer]
No data is shared. I used to include a crash reporter, but removed it for this very concern.
